recipes-wpe/thunder/include/thunder.inc

```diff
@@ -1,8 +1,8 @@
 # WPE Framework common
 HOMEPAGE = "https://github.com/WebPlatformForEmbedded"
-SECTION = "wpe"
+SECTION = "thunder"
```

Comment:
this in not known subsystem maybe call it plugins or some such

Reply:
Okay, plugins sounds great. I dont know what else to fill in here.